utils/mongo.js

```diff
@@ -1,21 +1,23 @@
-const { MongoClient } = require('mongodb');
+const mongoose = require("mongoose");
 
-const url = process.env.MONGO_DB_URL || 'mongodb+srv://admin:admin@cluster0.gt5ch.gcp.mongodb.net/'; // 'mongodb://localhost:27017'; 
+const url =
+  process.env.MONGO_DB_URL ||
```

Comment:
let's keep db name in separate constant and append to this url.

Reply:
we can do that but in my point of view if we are using single database in single application then threadpool management per database is much more suggestable anyhow if we need different database in picture then we should do change the nomenclature and provide different threadpool for the same